### ISSUE TITLE
Fatal Exception: java.security.ProviderException: Key size not available

### ISSUE THREAD

**Issue by fanjavaid** [talsec/Free-RASP-Android#26]
**Describe the bug**

**com.aheaditec.talsec.security.n.a**
**java.security.ProviderException - Failed to obtain information about key. Keystore error: 4**

```
Fatal Exception: java.security.ProviderException: Key size not available
       at android.security.keystore2.AndroidKeyStoreSecretKeyFactorySpi.getKeyInfo(AndroidKeyStoreSecretKeyFactorySpi.java:215)
       at android.security.keystore2.AndroidKeyStoreSecretKeyFactorySpi.engineGetKeySpec(AndroidKeyStoreSecretKeyFactorySpi.java:70)
       at javax.crypto.SecretKeyFactory.getKeySpec(SecretKeyFactory.java:562)
       at com.aheaditec.talsec.security.n.a(SourceFile:7)
       at com.aheaditec.talsec.security.n.m(SourceFile:2)
       at com.aheaditec.talsec.security.p.a(SourceFile:1)
       at com.aheaditec.talsec.security.f1.c(SourceFile:9)
       at com.aheaditec.talsec.security.f1.a(SourceFile:5)
       at com.aheaditec.talsec.security.e1.c(SourceFile:1)
       at com.aheaditec.talsec.security.e1.a(SourceFile:1)
       at com.aheaditec.talsec.security.a2.d(SourceFile:3)
       at java.lang.Thread.run(Thread.java:920)
```

**To Reproduce**
Can't reproduce the issue

**Expected behavior**
No crash is occured

**Screenshots**
-

**Please complete the following information:**
 - Device: Samsung S10+
 - OS version: Android 12
 - Version of freeRASP: 8.3.0

**Additional context**
-

**Comment by fanjavaid** [comment]
And maybe related to error below:

**com.aheaditec.talsec.security.n.a
java.lang.NullPointerException - Attempt to invoke interface method 'android.os.IBinder android.security.keystore.IKeystoreService.asBinder()' on a null object reference**

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
       at com.aheaditec.talsec.security.o.g(SourceFile:9)
       at com.aheaditec.talsec.security.m.f(SourceFile:1)
       at com.aheaditec.talsec.security.m.e(SourceFile:1)
       at com.aheaditec.talsec.security.n.m(SourceFile:1)
       at com.aheaditec.talsec.security.n.a(SourceFile:1)
       at com.aheaditec.talsec.security.f1.c(SourceFile:9)
       at com.aheaditec.talsec.security.f1.a(SourceFile:5)
       at com.aheaditec.talsec.security.e1.c(SourceFile:1)
       at com.aheaditec.talsec.security.e1.a(SourceFile:1)
       at com.aheaditec.talsec.security.a2.d(SourceFile:3)
       at java.lang.Thread.run(Thread.java:919)
```

**Comment by msikyna** [comment]
Hello @fanjavaid ,

thank you for reporting the issue! We will look at the issue!

Kind regards,
Talsec team

**Comment by msikyna** [comment]
Hello @fanjavaid , 
the issue has been fixed in the new release: [v9.0.0](https://github.com/talsec/Free-RASP-Android/releases/tag/v9.0.0).

Kind regards,
Talsec team
